CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(CMAKE_MODULE_PATH "$ENV{CARMA_OPT_DIR}/cmake")
 set(CMAKE_CXX_STANDARD 17)
 
 
-project(carma_clock
```

Comment:
How this break the Cmake config file? Do we have a screenshot or issue report of this?

Reply:
It did not break the build, it broke the Cmake config so when installed other projects could not find the library even when provided the CMake config.

Reply:
Looking back at it, it is likely because changing from `carma-clock` to 'carma_clock` also changes the library name in cmake and so it was looking for `carma-clockConfig.cmake` instead of `carma_clockConfig.cmake` but updating that in all the downstream users of this library is definitely unnecessary since we have solved the problem that the `-` character raised when installing this as a python module.